plugins/mcreator-localization/help/default/entity/do_ranged_attacks.md

```diff
@@ -1,3 +1,7 @@
 If checked, your entity will attack with a ranged item (similar to the Skeleton). 
 
-You can select your own ranged item from the list or use default (arrow).
\ No newline at end of file
+You can select your own ranged item from the list or use default (arrow).
+
+You can also select:

```

Comment:
Markdown will put all those in a single line. Either use list or double spacing to format this nicely